charts/prometheus-prefect-exporter/values.yaml

```
@@ -87,8 +87,8 @@ prometheusRule:
   #   labels:
   #     severity: critical
   #   annotations:
-  #     summary: Flow Run {{ $labels.flow_name }} {{ $labels.state_name }} (Flow {{ $labels.flow_name }})
-  #     description: "Flow Run failed or crashed\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
+  #     summary: Prefect server is down
+  #     description: The total number of Prefect server pods in the Prefect namespace is zero
```

```suggestion
  #     description: There are no Prefect server pods in the "prefect" namespace
```

Just a slight rephrase for clarity, and downcases + quotes the namespace name. Feel free to tweak this as needed 👍🏼